### PR TITLE
feat(security): Add redis config file related envs

### PR DESCRIPTION
**Pull request by jim-wang-intel**
New env added for redis' conf files related

Closes: #402

Signed-off-by: Jim Wang <yutsung.jim.wang@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/developer-scripts/blob/master/.github/Contributing.md.

## What is the current behavior?

No redis conf file used in the current behavior

## What is the new behavior?
Redis conf envs added

## Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?
This PR depends on https://github.com/edgexfoundry/edgex-go/pull/3115 and thus should be merged after that PR is merged.

## Other information

### CODE REVIEW

File: compose-builder/add-security.yml [hunk 1]
```diff
@@ -54,8 +55,13 @@ services:
       - common-sec-stage-gate.env
     environment:
       SECRETSTORE_TOKENFILE: /tmp/edgex/secrets/edgex-security-bootstrap-redis/secrets-token.json
+      DATABASECONFIG_PATH: /redis/conf
+      DATABASECONFIG_NAME: redis.conf
+    tmpfs:
+      - /redis
```
Comment (bnevis-i) [inline compose-builder/add-security.yml:61]: Suggest something more standards compliant like tmpfs: /run and put the redis conf there.

Comment (jim-wang-intel) [reply compose-builder/add-security.yml:61]: good point, will change it to /run

**Review by lenny-intel — COMMENTED** [review]
Looks good, approval pending response to @bnevis-i comment.

File: compose-builder/add-security.yml (hunk 1 above)
Comment (jim-wang-intel) [reply compose-builder/add-security.yml]: fixed

**Review by bnevis-i — APPROVED** [review]
LGTM

**Review by lenny-intel — APPROVED** [review]

**Review by lenny-intel — APPROVED** [review]
LGTM

**Comment by jinlinGuan** [comment]
recheck

**Pull request merged**